bin/buildBlueprints.js

```diff
@@ -111,6 +112,18 @@ if (argv.watch) {
   extensions.watch = true;
 }
 
+function addCacheBustedFile(name, m) {
```

Comment:
Just a question, how do we feel about JSDocs? When I was managing at HM, that was probably the number 1 request from new devs and after some initial resistance, I actually started to appreciate them a lot.